### PR TITLE
remove c.testcaselayer. with p.a.testing it is not needed any longer

### PR DESCRIPTION
**Pull request by tomgross**

### CODE REVIEW

File: checkouts.cfg [hunk 1]
```diff
@@ -41,6 +44,7 @@ auto-checkout =
     plone.supermodel
     plone.testing
     plonetheme.barceloneta
+    plonetheme.sunburst
```
Comment (tisto) [inline checkouts.cfg]: @tomgross Why has plonetheme.sunburst been added here?

**Comment by tomgross** [comment]
I think this is no longer needed, but p.a.imaging is. and some tests are failing there :(

**Comment by jensens** [comment]
Nice cleanup in the plone testing dependency hell. So do I understand correctly: 
- plonetheme.sunburst can be removed
- p.a.imaging needs some love - cecause it uses collective.testlayer? 
  Or am I wrong?

**Comment by tomgross** [comment]
No p.a.imageing has been ported to p.a.testing too. But there is a dependency import from p.a.imaging to p.a.blob. I'm currently porting p.a.blob to p.a.testing. After this is finished this PR can safely be merged.

Maybe this is more clear:

p.a.sunburst (ported to p.a.testing): DONE and released -> checkout can be removed
p.a.imaging (ported to p.a.testing): DONE -> checkout is needed
p.a.blob (ported to p.a.testing): WIP (dependency on p.a.imaging). I'm currently working on this one.

**Comment by tomgross** [comment]
After https://github.com/plone/plone.app.blob/pull/10 is merged this should be ready to go

**Pull request merged**